turbo.json

```diff
@@ -31,7 +31,7 @@
       "persistent": true,
       "cache": false
     },
-    "test": { "dependsOn": ["compile", "//#playwright-install"] },
+    "test": { "dependsOn": ["compile", "//#playwright-install"], "cache": false },
     "test:rust": { "dependsOn": ["download-keys", "compile"] },
```

Comment:
`cache": false` for `test:rust` necessary? otherwise LGTM

Reply:
the rust tests are very different. also they take an eternity in CI so i would very much like to cache them when possible.

they seem reliable so far, but i added some config to encourage turbo to run on crate change.

Reply:
is this repo using the faster runners like the monorepo? that could help

Reply:
buildjet 16vcpu on everything that uses cargo